perun-core/src/main/java/cz/metacentrum/perun/core/bl/ModulesUtilsBl.java

```diff
@@ -376,16 +405,19 @@
 	Map<String, String> transferQuotasBackToAttributeValue(Map<String, Pair<BigDecimal, BigDecimal>> transferedQuotasMap, boolean withMetrics) throws InternalErrorException;
 
 	/**
-	 * Merge two transfered Quotas together.
+	 * Merge two transferred Quotas together.
 	 * Paths are always unique, quotas are merged. (soft together and hard together)
 	 * Together means = bigger is better, 0 means unlimited so it is always bigger
```

Comment:
Now it should be according to fixed priorities. First which have the value set from following list will be used: MR:D:quotaOverride, MR:D:quota, R:D:defaultQuota.

Please update the implementation accordingly.